src/commands/CommandHandler.ts

```diff
@@ -185,20 +199,25 @@ export async function handleCommand(roomId: string, event: { content: { body: st
                 "!mjolnir alias remove <room alias>                                  - Deletes the room alias from whatever room it is attached to\n" +
                 "!mjolnir resolve <room alias>                                       - Resolves a room alias to a room ID\n" +
-                "!mjolnir since <date>/<duration> <action> <limit> [rooms...] [reason] - Apply an action ('kick', 'ban', 'mute', 'unmute' or 'show') to all users who joined a room since <date>/<duration> (up to <limit> users)\n" +
-                "!mjolnir shutdown room <room alias/ID> [message]                    - Uses the bot's account to shut down a room, preventing access to the room on this server\n" +
-                "!mjolnir powerlevel <user ID> <power level> [room alias/ID]         - Sets the power level of the user in the specified room (or all protected rooms)\n" +
-                "!mjolnir make admin <room alias> [user alias/ID]                    - Make the specified user or the bot itself admin of the room\n" +
-                "!mjolnir suspend <user ID>                                          - Suspend the specified user\n" +
-                "!mjolnir unsuspend <user ID>                                        - Unsuspend the specified user\n" +
-                "!mjolnir ignore <user ID/server name>                               - Add user to list of users/servers that cannot be banned/ACL'd. Note that this does not survive restart.\n" +
-                "!mjolnir ignored                                                     - List currently ignored entities.\n" +
+                "!mjolnir shutdown room <room alias/ID> [message]                    - Uses the bot's account to shut down a room, preventing access to the room on this server\n";
+
+            const botMenu =
+                "" +
+                "!mjolnir                                                            - Print status information\n" +
+                "!mjolnir status                                                     - Print status information\n" +
+                "!mjolnir verify                                                     - Ensures Mjolnir can moderate all your rooms\n" +
                 "!mjolnir help                                                       - This menu\n";
-            const html = `<b>Mjolnir help:</b><br><pre><code>${htmlEscape(menu)}</code></pre>`;
-            const text = `Mjolnir help:\n${menu}`;
+
+            const html = `<h3>Mjolnir help menu:</h3><br><b>Protection Actions/Options:</b><pre><code>${htmlEscape(protectionMenu)}</code></pre><br><b>Moderation Actions:</b><pre><code>${htmlEscape(actionMenu)}</code></pre><br><b>Policy List Options/Actions:</b><pre><code>${htmlEscape(policyListMenu)}</code></pre><br><b>Room Managment:</b><pre><code>${htmlEscape(roomsMenu)}</code></pre><br><b>Bot Status and Management:</b><pre><code>${htmlEscape(botMenu)}</code></pre>`;
```

Comment:
non-blocking: fwiw, the tilde character strings support newlines to retain formatting. For plaintext this is somewhat of an issue, but for HTML the extra whitespace doesn't affect anything. To make this a bit easier to read we can add a bunch of newlines:
```suggestion
            const html = `<h3>Mjolnir help menu:</h3><br>
            <b>Protection Actions/Options:</b><pre><code>${htmlEscape(protectionMenu)}</code></pre><br>
            <b>Moderation Actions:</b><pre><code>${htmlEscape(actionMenu)}</code></pre><br>
            <b>Policy List Options/Actions:</b><pre><code>${htmlEscape(policyListMenu)}</code></pre><br>
            <b>Room Managment:</b><pre><code>${htmlEscape(roomsMenu)}</code></pre><br>
            <b>Bot Status and Management:</b><pre><code>${htmlEscape(botMenu)}</code></pre>`;
```

The linter may have opinions on my formatting here.

(also `<br />` should be self-closing, like so)